### PR TITLE
[VSC-1510] rm old docs update links

### PR DESCRIPTION
**Pull request by brianignacio5**
## Description

* Remove GitHub documentation in favor of Espressif documentation.
* Update documentation links in README.md
* Add basic setup for `zh_CN` language build and update CI to include both.

## Type of change

- This change requires a documentation update

## Steps to test this pull request

Open Preview documentation link in this PR. Review documentation in Chinese.

Test readme links and extension links are updated.

## How has this been tested?

Test by click links in read and extension links. Check preview documentation that zh_CN is generated.

**Test Configuration**:
* ESP-IDF Version: Not relevant
* OS (Windows,Linux and macOS): Not related

## Checklist
- [ ] PR Self Reviewed
- [ ] Applied Code formatting
- [ ] Added Documentation
- [ ] Added Unit Test
- [ ] Verified on all platforms - Windows,Linux and macOS

### CODE REVIEW

**Review by Shen765 — APPROVED** [review]

**Review by radurentea — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -613,7 +605,7 @@ If you can't resolve the error, please search in the [github repository issues](
 
 # Code of Conduct
 
-This project and everyone participating in it is governed by the [Code of Conduct](./docs/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [vscode@espressif.com](mailto:vscode@espressif.com).
+This project and everyone participating in it is governed by the [Code of Conduct](./CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [vscode@espressif.com](mailto:vscode@espressif.com).
```
Comment (Fabricio-ESP) [inline README.md:608]: Make sure to move the CODE_OF_CONDUCT file to correct location, I don't see it in the git diff.

**Review by Fabricio-ESP — CHANGES_REQUESTED** [review]
All good besides the broken code of conduct link.

**Comment by Fabricio-ESP** [comment]
@brianignacio5 
Additional to your changes, There are some broken links in the "All Available commands" table, making references to files deleted in this PR. PTAL

File: README.md (hunk 1 above)
Comment (Fabricio-ESP) [reply README.md]: Fixed on commit: [b6f1eff](https://github.com/espressif/vscode-esp-idf-extension/pull/1342/commits/b6f1effc9bb4f160fbd2f56aec7032b88ca8828b)

**Review by Fabricio-ESP — APPROVED** [review]
All looks good now.

**Pull request merged**